src/journal/journal-internal.h

```diff
@@ -32,7 +32,7 @@ struct Match {
         /* For concrete matches */
         char *data;
         size_t size;
-        le64_t le_hash;
+        le64_t hash;
```

Comment:
Is this the correct type to use then? Also, I think that you mean HE (host endian) and not NE (network endian) in the commit message.

Reply:
good point